packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:math';
 
+import 'package:devtools_app/src/ui/fake_flutter/_real_flutter.dart';
```

Comment:
auto imports are doing the wrong thing. just go ahead and import 
'package:flutter/material.dart'